draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -174,67 +165,157 @@
    cannot require that such changes be observable beyond the scope
    of a single response.
 </t>
+</section>
+
+<section title="Evolution" anchor="intro.evolution">
 <t>
-   Each HTTP message is either a request or a
-   response. A server listens on a connection for a request, parses each
-   message received, interprets the message semantics in relation to the
-   identified target resource, and responds to that request with one or more
-   response messages. A client constructs request messages to communicate
-   specific intentions, examines received responses to see if the
-   intentions were carried out, and determines how to interpret the results.
+   HTTP has been the primary information transfer protocol for the World Wide
+   Web since its introduction in 1990. It began as a trivial mechanism for
+   low-latency requests, with a single method (GET) to request transfer of a
+   presumed hypertext document identified by a given pathname (HTTP/0.9). As
+   the Web grew, HTTP was extended to enclose requests and responses within
+   messages, transfer arbitrary data formats using MIME-like media types, and
+   route requests through intermediaries, eventually being defined as HTTP/1.0
+   <xref target="RFC1945"/>.
+</t>
+<t>
+   HTTP/1.1 was designed to refine the protocol's features while retaining
+   compatibility with the existing text-based messaging syntax, improving
+   its interoperability, scalability, and robustness across the Internet.
+   This included length-based payload delimiters for both fixed and dynamic
+   (chunked) content, a consistent framework for content negotiation,
+   opaque validators for conditional requests, cache controls for better
+   cache consistency, range requests for partial updates, and default
+   persistent connections. HTTP/1.1 was introduced in 1995 and published on
+   the standards track in 1997 <xref target="RFC2068"/>,
+   1999 <xref target="RFC2616"/>, and 2014
+   (<xref target="RFC7230"/> – <xref target="RFC7235"/>).
+</t>
+<t>
+   HTTP/2 (<xref target="RFC7540"/>) introduced a multiplexed session layer
+   on top of the existing TLS and TCP protocols for exchanging concurrent
+   HTTP messages with efficient header field compression and server push.
+   HTTP/3 (<xref target="HTTP3"/>) has gone even further, guiding the
+   development of QUIC as a secure multiplexing transport over UDP.
```

Comment:
I wouldn't have said "guiding" here.  Though it isn't *wrong*, it is probably better to talk about the outcomes of the process, which is that HTTP/3 provides greater independence for concurrent messages by using a QUIC, which is a secure multiplexed transport that uses UDP rather than TCP.